ui/app/templates/components/mfa/method-form.hbs

```diff
@@ -28,6 +29,7 @@
   @isActive={{this.isEditModalActive}}
   @confirmText={{@model.type}}
   @onConfirm={{perform this.save}}
+  @testSelector="mfa"
```

Comment:
Is this necessary? We should instead follow our convention of using `data-test-*` selectors for test selectors

Reply:
This Confirmation model component uses testSelector value and prefixes it with data-test-confirmation-modal.